examples/scene/volume_clipping.py

```diff
@@ -50,30 +46,6 @@
 axis.transform = affine
 
 
-# create colormaps that work well for translucent and additive volume rendering
```

Comment:
@brisvag , is it okay to remove these lines?

Reply:
Seems like it! Leftovers from my ugly copy-paste :P